### ISSUE TITLE
Unable to import azurerm_app_configuration_key which contains slashes e.g. "my/super/key"

### ISSUE THREAD

**Issue by tasiorek27** [terraform-providers/terraform-provider-azurerm#27172]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave comments along the lines of "+1", "me too" or "any updates", they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment and review the [contribution guide](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/README.md) to help.

### Terraform Version

1.8.5

### AzureRM Provider Version

4.0.0

### Affected Resource(s)/Data Source(s)

hashicorp/azurerm

### Terraform Configuration Files

```hcl
terraform {
  required_providers {
    azurerm = {
      source  = "hashicorp/azurerm"
      version = "4.0.0"
    }
}

resource "azurerm_app_configuration_key" "test" {
  configuration_store_id = azurerm_app_configuration.app_config.id
  key                    = "mm/test/abcz"
  value                  = "abc"

}

```
```


### Debug Output/Panic Output

```shell
Debug not needed.
```

### Expected Behaviour

The azurerm_app_configuration_key  should be imported.

### Actual Behaviour

```
terraform import azurerm_app_configuration_key.test  "https://some-app-config.azconfig.io/kv/mm/test/abcz?label="
╷
│ Error: parsing "https://some-app-config.azconfig.io/kv/mm/test/abcz?label=": AppConfiguration Nested Item should contain 2 segments, got 4 from "kv/mm/test/abcz"
│ 
│ 


```
If I will replace slashes with http encoded `%2F` then it will proceed but will fail as there is no such key in App Conf.

**Comment by teowa** [comment]
Hi @tasiorek27 , thanks for submitting this!
I can successfully import by replacing the `/` to `%2F` using the latest `4.0.1` version of the provider, so could it be the App Configuration Key doesn't exist on Azure ?
```hcl
resource "azurerm_app_configuration_key" "test" {
  configuration_store_id = azurerm_app_configuration.appconf.id
  key                    = "app/b/c"
  value                  = "a test"
}


```
![image](https://github.com/user-attachments/assets/846f5d5e-76c1-478a-8257-25fdb015882d)

**Comment by tasiorek27** [comment]
Yeah, it does work with this workaround. What's interesting is that slashes are supported in `labels` but not in `key` name. Are you going to work on it?

**Comment by rcskosir** [comment]
Thank you for taking the time to raise this! I am going to close this with @teowa‘s response as an answer. If you have future questions, I suggest using the [Community Resources](https://www.terraform.io/community), such as the [Azure Provider forum](https://discuss.hashicorp.com/c/terraform-providers/tf-azure/34).
